src/ui/units/datasets/components/DatasetTable/components/BatchActionPanel/components/DialogChangeDatasetFields/DialogChangeDatasetFields.scss

```diff
@@ -0,0 +1,23 @@
+.dl-dataset-dialog-change-fields {
+    &__content {
+        margin-top: var(--g-spacing-4);
+        display: flex;
+        gap: var(--g-spacing-6);
+        align-items: center;
+        justify-content: space-between;
+
+        & .dl-data-type-icon {
+            color: var(--g-color-text-secondary);
+        }
+    }
+
+    &__select {
+        flex: 1;
+    }
+
+    &__detail-btn {
+        .g-button__text {
```

Comment:
We should not use class names of the external libraries in our selectors

Reply:
Removed `&__detail-btn` class, because it an extra class that was needed at the design stage